### ISSUE TITLE
Expected more than 1 value per channel when training - On pspnet

### ISSUE THREAD

**Issue by vcartillier3** [meetshah1995/pytorch-semseg#214]
It seems Batchnorm module won't take a batch of size 1. 
You can check https://github.com/pytorch/pytorch/issues/4534

So during training if your last batch only has one element it was raise this error. Adding the following as a workaround:

in `train.py`:
```
if not images.size()[0]>1:
       continue
```

pytorch-1.1.0

**Comment by Da-He** [comment]
> It seems Batchnorm module won't take a batch of size 1.
> You can check [pytorch/pytorch#4534](https://github.com/pytorch/pytorch/issues/4534)
> 
> So during training if your last batch only has one element it was raise this error. Adding the following as a workaround:
> 
> in `train.py`:
> 
> ```
> if not images.size()[0]>1:
>        continue
> ```
> 
> pytorch-1.1.0

Exactly. Thanks!
